src/main/scala/com/fulcrumgenomics/vcf/AssessPhasing.scala

```diff
@@ -271,14 +268,11 @@ class AssessPhasing
                                        contig: String,
                                        contigLength: Int,
                                        intervalList: Option[IntervalList] = None): Iterator[VariantContext] = {
-    import scala.collection.JavaConversions.asScalaBuffer
-    val sampleName = reader.getFileHeader.getSampleNamesInOrder.head
+    val sampleName = reader.getFileHeader.getSampleNamesInOrder.iterator().next()
```

Comment:
I know, but there's really not a great alternative.  Could call `.get(0)` in this instance since it's a list, not a set.  But converting to scala just to call `.head` seems just as bad.